package.json

```diff
@@ -64,7 +64,7 @@
     "react-native-svg": "13.4.0",
     "react-query": "^3.39.2",
     "sentry-expo": "~6.0.0",
-    "zod": "^3.19.1"
+    "zod": "^3.20.0"
```

Comment:
I think this version bump turned out to be unnecessary - I thought I was going to use `z.coerce.date()` but ended up going a different way. I can revert if you want, but it seems like some nice new features were added.